unsigned-corim.cddl

```diff
@@ -2,9 +2,12 @@ unsigned-corim-map = {
   corim.id => $corim-id-type-choice
   corim.tags => one-or-more<$concise-tag-type-choice>
   ? corim.dependent-rims => one-or-more<corim-locator-map>
+  ? corim.profile => one-or-more<profile-type-choice>
```

Comment:
Do we really want profile to be a "Closed Type Choice" ?
I think, we can leave the flexibility in the interface to let it be extensible for a possible use expansion for future?

Reply:
I originally thought it should be $class-Id-type-choice. But we have vendor in class-map that resolves the namespace authority for uuids. Uri uses domain name to resolve ns Auth. And you can define a Uri that includes a uuid. So I don’t think it needs to be an extensible type choice.